source/renderer/app/api/api.js

```diff
@@ -352,7 +359,34 @@ export default class AdaApi {
     } catch (error) {
       Logger.debug('AdaApi::calculateTransactionFee error: ' + stringifyError(error));
       if (error.message === 'NotEnoughMoney') {
-        throw new NotEnoughFundsForTransactionFeesError();
+        const errorMessage = get(error, 'diagnostic.details.msg', '');
+        if (errorMessage.includes('Not enough coins to cover fee')) {
+          // Amount + fees exceeds walletBalance:
+          // - error.diagnostic.details.msg === 'Not enough coins to cover fee.'
+          // = show "Not enough Ada for fees. Try sending a smaller amount."
+          throw new NotEnoughFundsForTransactionFeesError();
+        } else if (errorMessage.includes('Not enough available coins to proceed')) {
+          const availableBalance = new BigNumber(
+            get(error, 'diagnostic.details.availableBalance', 0)
+          ).dividedBy(LOVELACES_PER_ADA);
+          if (walletBalance.gt(availableBalance)) {
+            // Amount exceeds availableBalance due to pending transactions:
+            // - error.diagnostic.details.msg === 'Not enough available coins to proceed.'
+            // - total walletBalance > error.diagnostic.details.availableBalance
+            // = show "Cannot calculate fees while there are pending transactions."
```

Comment:
Hmmm. Yes and no. I wonder if we couldn't teach users a bit here. It's not only about the fee, the funds just aren't available for making a transaction, regardless of the fee computation. 

Yet, if you think that it is better like it is from a UX perspective, then fine :)

Reply:
@KtorZ these error messages are displayed during transaction fee calculation so I think it is best to format them this way. The point is that you can not create a transaction without having transaction fees being calculated first. But I get your point too...